pom.xml

```diff
@@ -304,6 +304,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <maven.core.version>3.9.6</maven.core.version>
     <snappy-java.version>1.1.10.5</snappy-java.version>
     <hadoop-shaded-guava.version>1.2.0</hadoop-shaded-guava.version>
+    <com.nimbusds.nimbus-jose-jwt.version>9.24</com.nimbusds.nimbus-jose-jwt.version>
```

Comment:
Hadoop has upgraded to 9.31 in HADOOP-18711 due to the `json-smart` issue, and is now bumping further to 9.37.2 in HADOOP-19115 due to https://github.com/advisories/GHSA-gvpg-vgmx-xg6w.  Should we upgrade to 9.37.2 instead of 9.24?